ulwgl_run.py

```diff
@@ -270,12 +270,20 @@ def main() -> int:  # noqa: D103
         "ULWGL_ID": "",
     }
     command: List[str] = []
-    args: Union[Namespace, Tuple[str, List[str]]] = parse_args()
     opts: List[str] = None
+    # Expected files in this dir: pressure vessel, launcher files, runtime platform, runner, config
+    # root: Path = Path("/usr/share/ULWGL")
+    root: Path = Path(__file__).parent
```

Comment:
This might be too assumptious. The code doesn't check if the path actually is `/usr/share/ULWGL`, but all comments state so. As a packager, I could also imagine chosing `/usr/lib/ULWGL` or lower-case `ulwgl`. Would that be okay?

Reply:
Currently, the path for line 276 is used for quick testing purposes as I didn't want to package it to natively in my system and assumes the parent is the ULWGL-Launcher repository. Line 275 will be the real path that will be referred by the launcher once everything is ready. It's just commented at the moment and it doesn't serve as a comment toward line 276. 

As for package management, the launcher (ulwgl-run) is expected to be installed in `/usr/bin` then from there it'll reference the necessary Python modules and other relevant files involved in the setup operation 

cc @GloriousEggroll about paths and to clarify requirements

Reply:
We don't have any current documentation on packaging, however it's already been discussed and planned out within our development team.

```
packages and/or flatpak will do this:

Put everything in /usr/share/ULWGL, with a binary at /usr/bin/ulwgl-run

On first-run, /usr/bin/ulwgl-run will copy the required files from /usr/share/ULWGL to ~/.local/share/ULWGL/ and the steam tool files to ~/.local/share/steam/compatibilitytools.d

After that, any 3rd party tools can just run /usr/bin/ulwgl-run, which from that point forward will only do a version check to make sure the contents of /usr/share/ULWGL/ match that of the one in ~/.local/share/ULWGL
```